addons/web/static/src/scss/list_view.scss

```diff
@@ -122,6 +122,7 @@
         }
         .o_group_pager {
             position: relative;
+            width: 138px;
```

Comment:
Where does this number come from?

Reply:
IIRC it was the half of another element, but I can't find/remember which one it was